test/chains/elementChain.test.js

```diff
@@ -197,47 +197,47 @@ describe('Element chain', () => {
 	});
 
 	it('should convert to string with meaningful message', () => {
```

Comment:
I think assert.equal should be sufficient for string comparison, shouldn't it ?

Reply:
assert.equal is deprecated - "Stability: 0 - Deprecated: Use assert.strictEqual() instead."